packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -500,24 +492,19 @@ export const Slider = React.forwardRef((props, ref) => {
           {minValueLabel}
         </Box>
       )}
+      {tickMarkArea}
```

Comment:
was this intentionally moved here or can it be restored to the original position?

Reply:
there is a stacking issue where the ticks are appearing over the slider.

can be fixed using:
```scss
.iui-slider-container {
  // ...
  isolation: isolate;
}

.iui-slider-ticks {
  // ...
  z-index: -1;
}
```

Reply:
nit: I believe now the order of the `Slider`'s children ( `iui-slider-min`, `iui-slider`, `tickMarkArea`, `iui-slider-max` ) is different from the children in the slider html demo ( `iui-slider-min`, `tickMarkArea`, `iui-slider`, `iui-slider-max` ).

But this small mismatch can be ignored.